src/modules/flow/thingspeak/thingspeak.c

```diff
@@ -535,6 +536,7 @@ thingspeak_channel_update_open(struct sol_flow_node *node, void *data, const str
         return -ENOMEM;
     }
 
+    mdata->node = node;
```

Comment:
yeah, I know what you mean.
To be honest I believe the approach "mdata->node" is way more used in our code base than get_private_date.
We could change it at some point.